src/common/service-locator.ts

```diff
@@ -19,12 +23,32 @@ export function serviceLocator() {
       const privat24Handler = new BanksExchangeHandler(new Privat24Client());
       return monobankHandler.setNext(nbuHandler).setNext(privat24Handler);
     },
+    exchangerService: () => {
+      return new ExchangerService(serviceLocator().banksExchangeHandler());
+    },
+    ratesService: () => {
+      return new RatesService(new RatesRepository());
+    },
     emailService: async () => {
-      const kafkaProducer = new KafkaConsumer(await bootstrapKafka());
-      await kafkaProducer.connect();
-      const emailService = new EmailService(kafkaProducer);
+      const emailService = new EmailService(await serviceLocator().eventConsumer(), serviceLocator().ratesService());
       await emailService.start();
       return emailService;
     },
+    eventConsumer: async () => {
+      const kafkaConsumer = new KafkaConsumer(await bootstrapKafka());
```

Comment:
If I understand correctly, in this case, there will be several Kafka connects within one pod. For example, RabbitMQ supports sending and receiving messages simultaneously using one connect. We should think about reusing one Kafka connect